__mocks__/axios.ts

```diff
@@ -0,0 +1,57 @@
+const axios = require.requireActual('axios');
```

Comment:
mocking axios for tests

Reply:
[this](https://blog.kentcdodds.com/but-really-what-is-a-javascript-mock-10d060966f7d) was helpful to me, new to the idea of mocking deps. previously had us using [nock](https://github.com/node-nock/nock) to intercept network requests and mock responses.

Reply:
found [this](https://medium.com/@rickhanlonii/understanding-jest-mocks-f0046c68e53c) helpful as well

Reply:
if the `__mocks__` directory lives at the same level as `node_modules`, it'll automatically replace imported node modules w/ mock files of the same name. we can also configure jest to look elsewhere if we wanted to move `__mocks__` 👍